model/client4.go

```diff
@@ -632,6 +632,16 @@ func (c *Client4) UpdateTeamMemberRoles(teamId, userId, newRoles string) (bool,
 	}
 }
 
+// UpdateTeam will update a team
+func (c *Client4) UpdateTeam(teamId string, team *Team) (*Team, *Response) {
```

Comment:
No need to pass the teamId separately, it should be passed as part of the team struct